### PR TITLE
Updated ReadMe

### PR DESCRIPTION
**Pull request by InstincDev**
Updated ReadMe to include a description, screenshot, gif, and other sections. 

Thank you for the app and keeping it up and running!🍅😃

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,52 @@
...
+
+![mario_timer_screenshot](https://user-images.githubusercontent.com/67307808/175460423-74b26b1c-5119-4bbf-9317-628864108cfc.png)
+
+## Demo
+
+![ezgif com-gif-maker(1)](https://user-images.githubusercontent.com/67307808/175460667-d6c99b45-7eec-4432-b1d4-b3eda5479d61.gif)
+
+## Tech Stack
+
+**Client:** EJS
```
Comment (nmpereira) [inline README.md]: client and server both also have Socket.io

client has bootstrap 5

server has node-fetch, momentjs, MongoDB (mongoose)...among other things. Please link this to package.json on the project root

File: README.md [hunk 2]
```diff
@@ -1,9 +1,52 @@
...
+    1. clone `https://github.com/nmpereira/time-share`
+    2. install `node` and run `npm install`
+    3. Create an account in https://www.mongodb.com/ and create a collection, ensure network access from `0.0.0.0/0`
+    4. Copy the connection url from MongoDB to your local directory in a `.env` file. Name the variable `dbURI_time=<mongodb+srv://connection url>`
+```
+ 
+## Deployment
+
+
+To deploy this project run
```
Comment (nmpereira) [inline README.md]: "To run this project" (please reword this)

Also add a link to railway (this project is hosted on railway.app) https://railway.app?referralCode=mariojuanaaa

**Review by nmpereira — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (InstincDev) [reply README.md]: client and server list changes completed. link to package.json added.

File: README.md (hunk 2 above)
Comment (InstincDev) [reply README.md]: Deployment instructions reworded and railway link added.

**Pull request merged**